frontend/src/store/modules/place/actions.js

```diff
@@ -143,6 +143,96 @@ export default {
         });
     },
 
+    likePlaceReview: (context, data) => {
+        httpService.post(`reviews/${data.id}/like`)
+            .then( (res) => {
+                let review = context.state.places.find( (place) => { 
+                    return place.id === parseInt(data.placeId); 
+                }).review;
+                if (review.like === STATUS_NONE) {
+                    context.commit('SET_PLACE_REVIEW_LIKE_STATE', {
```

Comment:
context.commit('SET_PLACE_REVIEW_LIKE_STATE', {
            reviewId: reviewId,
            likeState: STATUS_LIKED
});

change `placeId: data.placeId`, to reviewId: data.id